rules/prefer-node-remove.js

```diff
@@ -46,34 +36,28 @@ const getArgumentName = arguments_ => {
 
 const create = context => {
 	return {
-		CallExpression(node) {
+		'CallExpression[callee.property.name="removeChild"]'(node) {
 			const {callee} = node;
 
-			if (node.arguments.length === 0 ||
+			if (node.arguments.length !== 1 ||
 				callee.type !== 'MemberExpression' ||
 				callee.computed
 			) {
 				return;
 			}
 
-			const methodName = getMethodName(callee);
 			const callerName = getCallerName(callee);
 
-			if (methodName === 'removeChild' && (
-				callerName === 'parentNode' ||
-				callerName === 'parentElement'
-			)) {
-				const argumentName = getArgumentName(node.arguments);
-
-				if (argumentName) {
-					const fix = isValueNotUsable(node) ? fixer => fixer.replaceText(node, `${argumentName}.remove()`) : undefined;
-
-					context.report({
-						node,
-						message: `Prefer \`${argumentName}.remove()\` over \`${callerName}.removeChild(${argumentName})\`.`,
-						fix
-					});
-				}
+			const argumentName = getArgumentName(node.arguments);
+
+			if (argumentName) {
+				const fix = isValueNotUsable(node) ? fixer => fixer.replaceText(node, `${argumentName}.remove()`) : undefined;
```

Comment:
I'm not sure, is there a `replaceNode` method? so we can 

1. replace `callee` with `arguments[0]`
2. replace callee.property with `remove`
3. remove `arguments[0]`

If we can do this, we can keep all comments, like

```js
/*comment1*/
foo
/*comment2*/
.removeChild
/*comment3*/
(
/*comment4*/
bar
/*comment5*/
)
```

Reply:
I checked the docs. seems not. this is not urgent.